docs/development/review.md

```diff
@@ -75,51 +129,52 @@ kindly,
-                                                  
-- Question: could you explain why this ended up having to be here?
-- Suggestion: you may have come across this already, but we might be able to use XYZ for this.
-- Change request: sorry, I appreciate the effort it took to get it this far. But the team 
-already agreed to solve this using XYZ. So I'm going to have to ask you to rework this. Let me know if I can help.
-
-
-A fourth type he sometimes uses is the reaction, which is not actionable, but might help to express emotions such as thanks.
-
-He further suggests to always keep things as clear as possible by using the following pattern:
-
->[type]: [actionable request] 
-
->[rationale or discussion]
-
-Whereas:
-- type: one of Question, Suggestion or Changes Request.
-- actionable request: is a short clear question or statement, which makes clear how the reviewee
-should respond.
-- rational or discussion: Possible further explanations of the reasoning behind the request.
-
-An example could be: 
->Suggestion: use XYZ. 
-
->You may have come across this already, but we might be able to use XYZ for this because blah blah blah etc.
-
-We adapted this slightly by bundling questions, suggestions and change requests in grouped listings and placing
-the reactions up front.
-
-## Inline Comments
-Github makes it very easy to leave inline comments. However, in the past, we observed, that large
-numbers of inline comments make a review hard to read and understand. We therefore propose
-only the leave inline comments for trivial directly actionable suggestions and change request, that
-do not need an rational (E.g. *Change Request: typo in word actionalbe*).
-
-Do not leave any reactions in inline comments. They tend to crowd the discussion without giving
-the possibility of being resolved (seens not actionable). Move them to the summary.
+Please commit the following file as part of the review to the root of the component.
 
+#### If the file already exists:
+* Please update the sections 'Maintenance' and 'Statistics'.
+* Please add the sections 'Criteria based Review for [ilias_version]' und 'Qualitative Review for [ilias_version]' **right below** the section 'Statistics'
 
+#### Review.md
 
+```
+# Review of `[name_of_component]` by [name_of_reviewer] at [date_of_review]
+
+## Maintenance
+* Maintainers/Coordinators:
+* Committers in last 365 days:
+
+## Statistics
+* Lines of Code: 
+* Nr of Tests: 
+* LoCs changed in last 365 days:
+* Nr of Commits in last 365 days: 
+* Nr of Open Mantis Issues: 
+* Mantis Issues resolved in last 365 days: 
+* Nr of Votes in last 365 days:
+* Nr of Votes for currently open issues:  
+
+## Criteria based Review for [ilias_version]
+
+* [ ] `[name_of_component]`and its administration can be used in devmode with PHP 8 and 8.1 without producing PHP-specific errors
+* [ ] The code style is PSR-12 + snake case compliant
```

Comment:
Maybe we should split this:

- [ ] The PHP code style complies with `PSR-12`.
- [ ] **No** new variables and class properties  are `camelCased`

Reply:
Are we sure we want to limit this to NEW variables and properties? It will be very hard to review this, right? I would have a tendency to go the tough route and say
-[ ] No variables and class properties are ```camelCased```
As this would state our goal even if not easy to attain, but I can actually live with many different versions of this rule.

Reply:
You are right. I am happy with that.

Reply:
Vielleicht lieber direkt auf unseren aktuellen Coding-Style verweisen? Dann müssen wir nicht hier immer hinterher ändern, wenn wir da was tun wollen.

Reply:
Auch wenn es mehr Pflegeaufwand bedeutet und die Info doppelt geführt wird, würde ich den eingeforderten Code-Style (oder für die jeweiligen Reviews wichtigen Punkte) hier explizit aufführen. So könnten wir auch neue oder präzisere Code-Style-Anforderungen pro Review-Durchgang mehr forcieren.

Reply:
Ich würde Dopplungen wo immer möglich vermeiden, erfahrungsgemäss schaffen wir das nicht die stellen dann auch nachzuziehen.

Reply:
Eine kleine Mehrheit scheint dafür sein nicht zu doppeln, eine Verlinkung auf den Coding-Style ergänzt.